app/.server/event-web-api/event-api.ts

```diff
@@ -21,25 +21,36 @@ export class EventApi {
 
     const search = new ProposalSearchBuilder(this.eventSlug, 'no-user', filters);
 
-    const proposals = await search.proposals({ reviews: false });
+    const proposals = await search.proposals();
 
     return {
       name: event.name,
+      startDate: event.conferenceStart,
+      endDate: event.conferenceEnd,
       proposals: proposals.map((proposal) => {
         return {
+          id: proposal.id,
           title: proposal.title,
           abstract: proposal.abstract,
+          deliberationStatus: proposal.deliberationStatus,
+          confirmationStatus: proposal.confirmationStatus,
+          publicationStatus: proposal.publicationStatus,
           level: proposal.level,
+          references: proposal.references,
           formats: proposal.formats.map((f) => f.name),
           categories: proposal.categories.map((c) => c.name),
           languages: proposal.languages as string[],
           speakers: proposal.speakers.map((speaker) => ({
             name: speaker.name,
             bio: speaker.bio,
             company: speaker.company,
+            references: speaker.references,
             picture: speaker.picture,
+            location: speaker.location,
+            email: speaker.email,
```

Comment:
I'd like to avoid exposing personal information like `email` in the Web API. If the API key is leaked, it could expose those data. 
If you need it, the email can be exported from the manual JSON export by event owners.

Reply:
When we organize a conference we need to be able to communicate to speakers outside of the conference web site. Data leak is an important problem, but the solution is not to limit the exposed data but to improve the safety of the APIs.